README.md

```diff
@@ -279,13 +281,20 @@ Vimscript:
   It can be installed using npm:
   [node-vimlint](https://www.npmjs.com/package/vimlint).
 
-  Or you could create a wrapper script ``vimlint`` and add it to your PATH:
+  Or you could create a wrapper script `vimlint` and add it to your PATH:
 
     ```sh
     #!/bin/sh
-    ~/Vcs/vim-vimlint/bin/vimlint.sh -l ~/Vcs/vim-vimlint -p ~/Vcs/vim-vimlparser "$@"
+    ~/Vcs/vim-vimlint/bin/vimlint.sh  \
+      -l ~/Vcs/vim-vimlint            \
+      -p ~/Vcs/vim-vimlparser         \
+      "$@"
```

Comment:
Because of the line length?

I don't think it is better then - I was just copy'n'pasting the output from mdl.

But feel free to leave it as-is: it makes it clearer what the args are.